kolibri/core/webpack/hooks.py

```diff
@@ -366,7 +359,7 @@ def render_to_page_load_sync_html(self):
         :return: HTML of script tags for insertion into a page.
         """
         tags = (['<script>var coreLanguageMessages = {messages};</script>'.format(
-            messages=self.frontend_messages)] if self.frontend_messages else []) + list(self.js_and_css_tags())
+            messages=self.frontend_messages())] if self.frontend_messages() else []) + list(self.js_and_css_tags())
```

Comment:
recommend breaking this up/reformatting for readability.

maybe something like this?

```python
        tags = []
        if self.frontend_messages():
            tags.append('<script>var coreLanguageMessages = {};</script>'.format(self.frontend_messages()))
        tags.extend(self.js_and_css_tags())
```